src/schemas/21-526EZ-allclaims/schema.js

```diff
@@ -0,0 +1,583 @@
+    },
+    attachments: {
+      type: 'array',
+      items: {
+        type: 'object',
+        required: ['name', 'attachmentId'],
+        properties: {
+          name: {
+            type: 'string'
+          },
+          confirmationCode: {
+            type: 'string'
+          },
+          attachmentId: {
+            type: 'string',
+            enum: documentTypes526.map(doc => doc.value),
+            enumNames: documentTypes526.map(doc => doc.label)
+          }
+        }
+      }
+    },
+    bankAccountType: {
+      type: 'string',
+      enum: ['Checking', 'Savings']
+    },
+    bankAccountNumber: {
+      type: 'string',
+      minLength: 4,
+      maxLength: 17
+    },
+    bankRoutingNumber: {
+      type: 'string',
+      pattern: '^\\d{9}$'
+    },
+    bankName: {
+      type: 'string',
+      maxLength: 35
+    },
+    isVAEmployee: {
+      type: 'boolean'
+    },
+    standardClaim: {
+      type: 'boolean',
+      default: false
+    },
+    ptsd781Attachment: {
+      $ref: '#/definitions/ptsdAttachment'
+    },
+    ptsd781aAttachment: {
+      $ref: '#/definitions/ptsdAttachment'
+    },
+    form4142: {
+      type: 'object',
+      properties: {
+        limitedConsent: {
+          type: 'string'
+        },
+        providerFacility: {
+          type: 'array',
+          required: [
+            'providerFacilityName',
+            'treatmentDateRange',
+            'providerFacilityAddress'
+          ],
+          items: {
+            type: 'object',
+            properties: {
+              providerFacilityName: {
+                type: 'string'
+              },
+              treatmentDateRange: {
+                $ref: '#/definitions/dateRange'
+              },
+              providerFacilityAddress: {
+                $ref: '#/definitions/address'
+              }
+            }
+          }
+        },
+        privacyAgreementAccepted: {
+          $ref: '#/definitions/privacyAgreementAccepted'
+        }
+      }
+    },
+    form0781: {
+      type: 'object',
+      incident: {
+        type: 'array',
+        items: {
+          type: 'object',
+          properties: {
+            personalAssault: {
+              type: 'boolean'
+            },
+            medalsCitations: {
+              type: 'string'
+            },
+            incidentDate: {
+              $ref: '#/definitions/date'
+            },
+            incidentLocation: {
+              type: 'string'
+            },
+            incidentDescription: {
+              type: 'string'
+            },
+            unitAssigned: {
+              type: 'string'
+            },
+            unitAssignedDates: {
+              $ref: '#/definitions/dateRange'
+            },
+            remarks: {
+              type: 'string'
+            },
+            personInvolved: {
+              type: 'array',
+              items: {
+                type: 'object',
+                name: {
+                  $ref: '#/definitions/fullName'
```

Comment:
Looks like this isn't in the definitions either.

Reply:
Looks like this slipped by on vets-website as well, because it isn't used anywhere. Since I don't know what their intent was (our MVP heavily modifies the fullName common def), I'm just going to add the common fullName def in here and they can change it later if needed.